src/lucky/skip_unique_route_check.cr

```diff
@@ -0,0 +1,6 @@
+# Include this in an action to skip route uniqueness checks.
+module Lucky::SkipUniqueRouteCheck
```

Comment:
Not sure that it's necessary to provide this. If you skip the compile time check, you're still going to get a runtime error from the duplicate checks in lucky_router https://github.com/luckyframework/lucky_router/blob/d6020f0c404e033abb920cf29e34ec8d752525c2/src/lucky_router/matcher.cr#L50

Reply:
Good point. We discussed this in chat and I think we can leave this here pre-1.0 (or at least for one version) since this uses different logic for checking duplicates. If it seems to not have issues then we can remove it!

Does that sound good?

Reply:
Created an issue to track this here: https://github.com/luckyframework/lucky/issues/1541